pipelinewise/cli/__init__.py

```diff
@@ -82,13 +83,18 @@ def main():
     # import_config : this is for backward compatibility; use 'import' instead from CLI
     if args.command == 'import' or args.command == 'import_config':
         if args.dir == '*':
-            print("You must specify a directory path with config YAML files using the argumant --dir")
+            print("You must specify a directory path with config YAML files using the argument --dir")
             sys.exit(1)
 
         # Every command argument is mapped to a python function with the same name, but 'import' is a
         # python keyword and can't be used as function name
         args.command = 'import_project'
 
+    if args.command == 'validate_config':
```

Comment:
can we change it to `args.command == 'validate'` to be in sync to the `commands` list?

Currently the `args.dir` check is not running when running without the required `--dir` arg:
(pipelinewise) root@be2ea33b59aa:/opt/pipelinewise# pipelinewise validate 
2019-10-29 14:56:03 INFO: Searching YAML config files in *
...FileNotFoundError: [Errno 2] No such file or directory: '*'

Reply:
@koszti right, sorry, totally forgot to change it!